lib/rules/order-in-models.js

```diff
@@ -1,55 +1,57 @@
 'use strict';
 
-var utils = require('../utils/utils');
 var ember = require('../utils/ember');
+var reportUnorderedProperties = require('../utils/property-order').reportUnorderedProperties;
+
+const ORDER = [
+  'attribute',
+  'relationship',
+  'single-line-function',
+  'multi-line-function',
+  'other',
+];
+
+const NAMES = {
+  'attribute': 'attribute',
+  'relationship': 'relationship',
+  'single-line-function': 'single-line function',
+  'multi-line-function': 'multi-line function',
+  'other': 'property',
```

Comment:
Okay, so let's leave this as a separate concern. Anyway your solution seems to be better than enough and I'm happy to push this forward :)